src/server/core/journey/route/graph.ts

```
@@ -144,260 +128,409 @@ function addCustomLocations(
 
-function connectOriginToStations(
+/**
+ * Find nearest stations to a location
+ */
+function findNearestStations(
   graph: Graph<NodeData, EdgeData>,
-  originId: string,
-  origin: Coordinates,
-  maxWalkingDistance: number
+  location: Coordinates,
+  limit: number = 5
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)